content_scripts/link_hints.coffee

```diff
@@ -288,8 +291,13 @@ class LinkHintsMode
 
     else
       @tabCount = previousTabCount if event.ctrlKey or event.metaKey or event.altKey
+      keyChar =
+        if Settings.get "filterLinkHints"
+          KeyboardUtils.getKeyChar(event)
+        else
+          KeyboardUtils.getKeyChar(event).toLowerCase()
```

Comment:
Not sure if duplicating `KeyboardUtils.getKeyChar(event)` was the most elegant solution, but it felt better than adding another variable just for keeping that value.

Reply:
What about
```
keyChar = KeyboardUtils.getKeyChar event
unless Settings.get "filterLinkHints"
  keyChar = keyChar.toLowerCase()
```

Reply:
Not a huge fan of mutation, but if you think that's better, I'll change it?